common/src/main/java/uk/gov/justice/services/common/converter/Converter.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.justice.services.common.converter;
+
+/**
+ * A converter converts a source object of type S to a target of type T.
+ */
+@FunctionalInterface
+public interface Converter<S, T> {
```

Comment:
Is this effectively recreating (albeit with a specific name) the 'java.util.function.Function' provided Java Functional Interface?